docker-compose.yml

```diff
@@ -372,6 +372,26 @@ services:
          - opentsdb.pendev
 
 
+  openkilda-gui:
```

Comment:
`openkilda-` prefix is excess here. Whole this file is for `open-kilda`. And docker-compose use name of directory where docker-compose.yml as `project name`...

Reply:
Its actually our project name `openkilda-gui`

Reply:
Is it a separate project? or part of some other project?

Reply:
It is a separate project under open-kilda like NB and FL.